config/core/300-resources/service.yaml

```diff
@@ -59,6 +59,1129 @@ spec:
   - << : *version
     name: v1
     storage: true
+    schema:
+      openAPIV3Schema:
+        x-kubernetes-preserve-unknown-fields: true
+        type: object
+        required:
+        - apiVersion
+        - kind
+        - metadata
+        properties:
+          apiVersion:
+            type: string
+            description: "APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values.
+              \nMore info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources"
```

Comment:
The new lines impact how it is rendered in some yaml tools. It looks better with it. Does it cause any issue beyond appearing odd in the raw form?

Reply:
I think the yaml pipe `|` should work to capture the new lines you want?

Reply:
That is very interesting. I didn't know about the use of the pipe as the `literal style` in yaml.
Thanks, that does improve readability.
This may take a bit to clean up.

Reply:
I used a pipe for all the descriptions to keep things consistent. The down side to the `literal style` is that it does not allow any escaping. Fortunately this does not appear to effect how I'm using it in the IDE.